redirect/nginx.conf

```diff
@@ -0,0 +1,13 @@
+worker_processes 1;
+daemon off;
+
+error_log <%= ENV["APP_ROOT"] %>/nginx/logs/error.log;
+events { worker_connections 1024; }
+
+http {
+  server {
+    listen <%= ENV["PORT"] %>;
+    server_name localhost;
```

Comment:
the listen header doesn't need 0.0.0.0 and having the single server {} makes this the default.